exchanges/btcmarkets/btcmarkets_test.go

```diff
@@ -18,489 +17,421 @@ const (
+	if err != nil {
+		t.Error(err)
 	}
 }
 
-func TestCancelExchangeOrder(t *testing.T) {
-	b.SetDefaults()
-	TestSetup(t)
-
-	if areTestAPIKeysSet() && !canManipulateRealOrders {
-		t.Skip("API keys set, canManipulateRealOrders false, skipping test")
+func TestGetTransactions(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip("API keys required but not set, skipping test")
 	}
-
-	currencyPair := currency.NewPair(currency.LTC, currency.BTC)
-
-	var orderCancellation = &order.Cancel{
-		OrderID:       "1",
-		WalletAddress: "1F5zVDgNjorJ51oGebSvNCrSAHpwGkUdDB",
-		AccountID:     "1",
-		CurrencyPair:  currencyPair,
+	_, err := b.GetTransactions("BTC")
+	if err != nil {
+		t.Error(err)
 	}
+}
 
-	err := b.CancelOrder(orderCancellation)
-	if !areTestAPIKeysSet() && err == nil {
-		t.Error("Expecting an error when no keys are set")
+func TestCreateNewReport(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip("API keys required but not set, skipping test")
 	}
-	if areTestAPIKeysSet() && err != nil {
-		t.Errorf("Could not cancel orders: %v", err)
+	_, err := b.CreateNewReport("TransactionReport", "json")
+	if err != nil {
+		t.Error(err)
 	}
 }
 
-func TestCancelAllExchangeOrders(t *testing.T) {
-	b.SetDefaults()
-	TestSetup(t)
-
-	if areTestAPIKeysSet() && !canManipulateRealOrders {
-		t.Skip("API keys set, canManipulateRealOrders false, skipping test")
+func TestGetReport(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip("API keys required but not set, skipping test")
 	}
-
-	currencyPair := currency.NewPair(currency.LTC, currency.BTC)
-
-	var orderCancellation = &order.Cancel{
-		OrderID:       "1",
-		WalletAddress: "1F5zVDgNjorJ51oGebSvNCrSAHpwGkUdDB",
-		AccountID:     "1",
-		CurrencyPair:  currencyPair,
+	_, err := b.GetReport("1kv38epne5v7lek9f18m60idg6")
+	if err != nil {
+		t.Error(err)
 	}
+}
 
-	resp, err := b.CancelAllOrders(orderCancellation)
-
-	if !areTestAPIKeysSet() && err == nil {
-		t.Error("Expecting an error when no keys are set")
-	}
-	if areTestAPIKeysSet() && err != nil {
-		t.Errorf("Could not cancel orders: %v", err)
+func TestRequestWithdaw(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip("API keys required but not set, skipping test")
 	}
-
-	if len(resp.Status) > 0 {
-		t.Errorf("%v orders failed to cancel", len(resp.Status))
+	_, err := b.RequestWithdraw("BTC", 1, "sdjflajdslfjld", "", "", "", "")
+	if err != nil {
+		t.Error(err)
 	}
 }
 
-func TestModifyOrder(t *testing.T) {
-	_, err := b.ModifyOrder(&order.Modify{})
-	if err == nil {
-		t.Error("ModifyOrder() Expected error")
+func TestBatchPlaceCancelOrders(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() || !canManipulateRealOrders {
+		t.Skip("skipping test, either api keys or manipulaterealorders isnt set correctly")
+	}
+	var pOrders []placeBatch
+	placeOrders := placeBatch{MarketID: BTCtoAUD,
+		Amount:    "11000",
+		Price:     "1",
+		OrderType: "Limit",
```

Comment:
When testing this exchange on v2, "limit" still works so you can use our built in order types and sides